app/src/AppRouter.tsx

```diff
@@ -25,6 +26,7 @@ function AppRouter() {
         <Route path="/schedule/timetable" element={[<ScheduleTimetable />]} />
         <Route path="/schedule/generate" element={[<ScheduleGenerate />]} />
         <Route path="/professorprofile" element={[<ProfessorProfile />]} />
+        <Route path="/ErrorPage" element={[<ErrorPage />]} />

```

Comment:
Let's also change this path from "/ErrorPage" to just "/error" to match the rest of the routes in this file